cpp/src/aztec/proof_system/composer/composer_base.hpp

```diff
@@ -56,8 +57,49 @@ struct poly_triple {
     barretenberg::fr q_r;
     barretenberg::fr q_o;
     barretenberg::fr q_c;
+
+    friend bool operator==(poly_triple const& lhs, poly_triple const& rhs)
+    {
+        // clang-format off
+        return
+            lhs.a == rhs.a &&
+            lhs.b == rhs.b &&
+            lhs.c == rhs.c &&
+            lhs.q_m == rhs.q_m &&
+            lhs.q_l == rhs.q_l &&
+            lhs.q_r == rhs.q_r &&
+            lhs.q_o == rhs.q_o &&
+            lhs.q_c == rhs.q_c;
+        // clang-format on
+    }
```

Comment:
Is it possible to replace this with `bool operator==(poly_triple const& lhs, poly_triple const& rhs) = default;`?

Reply:
If the minimum version of CPP supported is C++20, then yep we can use default comparisons here instead

Reply:
Checking the CMake file, it looks like C++20 is required, so I can make that change.